sub-packages/bionemo-evo2/README.md

```diff
@@ -0,0 +1,273 @@
+# bionemo-evo2
+
+`bionemo-evo2` is a `pip`-installable package that contains **data preprocessing**, **training**, and **inferencing** code for Evo2, a new `Hyena`-based foundation model for genome generation and understanding. Built upon `Megatron-LM` parallelism and `NeMo2` algorithms, `bionemo-evo2` provides the remaining tools necessary to effectively fine-tune the pre-trained Evo2 model checkpoint on user-provided sequences at scale, and generate state-of-the-art life-like DNA sequences from Evo2 for downstream metagenomic tasks.
+
+## Quickstart tutorials
+
+Please see
+
+## Installation
+
+To install this package, execute the following command:
+
+```bash
+pip install -e .
+```
+
+To run unit tests, execute the following command:
+
+```bash
+pytest -v .
+```
+
+## Preprocessing
+
+To train or fine-tune Evo2 on a custom dataset, we need to preprocess and index sequence data for training from raw FASTA files into tokenized binaries compliant with `NeMo2` / `Megatron-LM`. For more information about how to configure your data for training, refer to [data/README.md](src/bionemo/evo2/data/README.md) and [utils.config.Evo2PreprocessingConfig](src/bionemo/evo2/utils/config.py).
+
+```bash
+preprocess_evo2 -c <CONFIG_PATH>
+```
+
+## Training
+
+Given a preprocessed collection of preprocessed datasets, and optionally a pre-trained NeMo2 checkpoint for Evo2, training can be executed using the following command:
+
+```bash
+$ train_evo2 --help
```

Comment:
probably this section should be updated

Reply:
Updated in the new commit I'm about to push.